examples/ASDKgram/Sample/AppDelegate.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AppDelegate.h
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 2/16/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@protocol PhotoFeedViewControllerProtocol <NSObject>
```

Comment:
Please move this into the header file for the photo view controller. Then, you can import it only in the implementation file and do not need it in this header.